lib/elixir/lib/enum.ex

```diff
@@ -1341,8 +1341,8 @@ defmodule Enum do
   If multiple elements are considered maximal, the first one that was found
   is returned.
 
-  Calls the provided `empty_fallback` function and returns its value if
-  `enumerable` is empty. The default `empty_fallback` raises `Enum.EmptyError`.
+  Calls the provided `empty_fallback/0` function and returns its value if
```

Comment:
This is identifying an anonymous function that is given as argument, it should be `empty_fallback` so it is not confused with an eventual function named `empty_fallback/0` on the current module. We never identify anonymous functions by their arity, only their name.